_source/_posts/2022-08-15-spring-boot-crud-update.md

```diff
@@ -0,0 +1,1077 @@
+---
+layout: blog_post
+title: "Spring Boot CRUD app with Vue and Quasar"
+author: andrew-hughes
+by: contractor
+communities: [java,javascript]
+description: "Build a Spring Boot Todo app with a Vue client using Okta for security and the Quasar framework for client components"
```

Comment:
Match the same description as the previous one. You can improve it if you like, but it seems to have worked well so far.